addons/website_sale_autocomplete/controllers/main.py

```diff
@@ -62,9 +66,18 @@ def _guess_number_from_input(self, source_input, standard_address):
         guessed_house_number = guessed_house_number.split(',')[0].strip()
         return guessed_house_number
 
-    def _perform_place_search(self, partial_address, api_key=None, session_id=None, language_code=None, country_code=None):
+    def _get_api_place_api_key(self):
+        return request.env['website'].get_current_website().google_places_api_key
```

Comment:
If the plan is to override this method in another module, it is not safe enough, see discussion on task 2806842.

Reply:
yes, same that stripe, my bad.